bundles/azure/aso/psql/bundle/config/99-kapp-config.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kapp.k14s.io/v1alpha1
+kind: Config
+waitRules:
+- supportsObservedGeneration: false
+  ytt:
+    funcContractV1:
+      resource.star: |
+        def is_done(resource):
+            kind = resource.kind
+            name = resource.metadata.name
+
+            if not "status" in resource or not "conditions" in resource.status:
+              return {"done": False, "successful": False, "message": "Resource " + kind + "/" + name + " waiting for status"}
+            end
+
+            condition = resource.status.conditions[0]
```

Comment:
question: This expect the `Ready` condition to always be the first in the list. Is that guaranteed?

Reply:
As far as we can tell, there never is any other condition.